application/Config/Pager.php

```diff
@@ -19,7 +19,8 @@ class Pager extends BaseConfig
 	*/
 	public $templates = [
 		'default_full'   => 'CodeIgniter\Pager\Views\default_full',
-		'default_simple' => 'CodeIgniter\Pager\Views\default_simple'
+		'default_simple' => 'CodeIgniter\Pager\Views\default_simple',
+		'default_header' => 'CodeIgniter\Pager\Views\default_header'
```

Comment:
Maybe this can be hidden.

Calling 'CodeIgniter\Pager\Views\default_header' directly in [$pager->links](https://github.com/bcit-ci/CodeIgniter4/pull/622/files#diff-2838c7b5acb9a877d850b24f7fec94e0R356).